README.md

```diff
@@ -57,7 +58,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: **O(n^2)**
```

Comment:
This one is actually O(n*m) --> but truth be told I accidentally wrote the key with your answer the first time too. Know why O(n*m) is right?